scalding-core/src/main/scala/com/twitter/scalding/ExecutionUtil.scala

```diff
@@ -2,17 +2,27 @@ package com.twitter.scalding
 
 import com.twitter.algebird.Semigroup
 
+import scala.util.{Failure, Success, Try}
+
 object ExecutionUtil {
   /**
+   * Useful for when using parallelism and want to output the final state of the job
+   */
+  def liftedSequenceUnit[T](executions: Execution[Seq[Try[T]]], handleComplete: Try[T] => Unit): Execution[Unit] =
```

Comment:
It really is a side effect function, there are a few cases where I can see it being useful for alerting/logging failures especially on adhoc jobs. 

I can drop if you find the API confusing and move it internally into my code base.